src/Hedgehog/Extras/Stock/Time.hs

```diff
@@ -21,4 +20,4 @@ showUTCTimeSeconds time = show @Int64 (floor (DTC.utcTimeToPOSIXSeconds time))
 
 -- | Format the given time as an ISO 8601 date-time string
 formatIso8601 :: UTCTime -> String
-formatIso8601 = DT.formatTime DT.defaultTimeLocale (DT.iso8601DateFormat (Just "%H:%M:%SZ"))
+formatIso8601 = DT.formatTime DT.defaultTimeLocale "%Y-%m-%dT%H:%M:%SZ"
```

Comment:
Was this change needed to compile?

Reply:
Wasn't need to compile but `iso8601DateFormat` is deprecated. My replacement just inlines what that function doens.